src/theme.js

```diff
@@ -12,6 +12,8 @@ export const theme = {
         white: "#FFFFFF",
     },
     breakpoint: {
+        miniMobile: 568,
         mobileMax: 767,
+        ipad: 1024,
```

Comment:
Hej, ja bym się jeszcze zastanowił nad jakimś breakpointem, dla monitorów taki jak ja mam, czyli hd 1366x768, bo obecna wersja kontenera jest zbyt duża na takie ekrany. Obecna wersja jest dobra na fullhd.

Reply:
Dokładnie tak jak wojtek, prawdopodobnie będzie użyty do zmienienia ilości kolumn w filmach na 3 a w osobach na 4/5, ale przyda się

Reply:
Ogólnie to zmienna liczbę kolumn możemy zrobić podobnie do tego jak jest w kontenerze osób. Możesz Szymon zerknąć jak to tam śmiga, ogólnie w zależności od szerokości okna jest różna liczba osób w rzędzie. Ważniejesza do ustawienia jest jakaś pośrednia szerokość Wrappera, bo jak jest za duża to nie ma wgl paddingu po prawej i lewej stronie na moim ekranie.

Reply:
Dodałam :D czy teraz będzie dobrze?